opm/simulators/flow/ExtraConvergenceOutputThread.cpp

```diff
@@ -134,7 +175,21 @@ namespace {
             }
 
             os << std::right << std::setw(colSize + 1)
-               << (report.wellFailed() ? "FAIL" : "CONV") << '\n';
+               << (report.wellFailed() ? "FAIL" : "CONV");
+            if (report.wellFailed()) {
+                for (const auto& wf : report.wellFailures()) {
+                    os << " { "
+                       << wf.wellName() << " "
+                       << wellFailureTypeToString(wf.type());
+                    if (wf.type() == Opm::ConvergenceReport::WellFailure::Type::MassBalance) {
+                        os << " "
+                           << severityToString(wf.severity()) << "Severity "
+                           << "Phase " << wf.phase();
```

Comment:
Here too, something like
```C++
"Phase=" << wf.phase()
```
will make the format _slightly_ more machine readable.